library/nextcloud

```diff
@@ -3,92 +3,47 @@
 Maintainers: Nextcloud <docker@nextcloud.com> (@nextcloud)
 GitRepo: https://github.com/nextcloud/docker.git
 
-Tags: 14.0.14RC1-apache, 14.0.14-rc-apache, 14.0-rc-apache, 14-rc-apache, 14.0.14RC1, 14.0.14-rc, 14.0-rc, 14-rc
+Tags: 14.0.14-apache, 14.0-apache, 14-apache, 14.0.14, 14.0, 14
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 14.0-rc/apache
-
-Tags: 14.0.14RC1-fpm-alpine, 14.0.14-rc-fpm-alpine, 14.0-rc-fpm-alpine, 14-rc-fpm-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 14.0-rc/fpm-alpine
-
-Tags: 14.0.14RC1-fpm, 14.0.14-rc-fpm, 14.0-rc-fpm, 14-rc-fpm
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 14.0-rc/fpm
-
-Tags: 14.0.13-apache, 14.0-apache, 14-apache, 14.0.13, 14.0, 14
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: 060cf0883ff12241081778714507e2823d84e629
+GitCommit: 7ec56a6567044cc5b70adefe85f613a643cb59a9
 Directory: 14.0/apache
 
-Tags: 14.0.13-fpm-alpine, 14.0-fpm-alpine, 14-fpm-alpine
+Tags: 14.0.14-fpm-alpine, 14.0-fpm-alpine, 14-fpm-alpine
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: ea8fae60eb7bb6839ea551132a3e067e1f3e7fdf
+GitCommit: 7ec56a6567044cc5b70adefe85f613a643cb59a9
 Directory: 14.0/fpm-alpine
 
-Tags: 14.0.13-fpm, 14.0-fpm, 14-fpm
+Tags: 14.0.14-fpm, 14.0-fpm, 14-fpm
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: 060cf0883ff12241081778714507e2823d84e629
+GitCommit: 7ec56a6567044cc5b70adefe85f613a643cb59a9
 Directory: 14.0/fpm
 
-Tags: 15.0.11RC1-apache, 15.0.11-rc-apache, 15.0-rc-apache, 15-rc-apache, 15.0.11RC1, 15.0.11-rc, 15.0-rc, 15-rc
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 15.0-rc/apache
-
-Tags: 15.0.11RC1-fpm-alpine, 15.0.11-rc-fpm-alpine, 15.0-rc-fpm-alpine, 15-rc-fpm-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 15.0-rc/fpm-alpine
-
-Tags: 15.0.11RC1-fpm, 15.0.11-rc-fpm, 15.0-rc-fpm, 15-rc-fpm
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
-GitCommit: a4efb0e686ade89c0dfdf9a1fe30f77e3099e0a7
-Directory: 15.0-rc/fpm
-
-Tags: 15.0.10-apache, 15.0-apache, 15-apache, production-apache, 15.0.10, 15.0, 15, production
+Tags: 15.0.11-apache, 15.0-apache, 15-apache, 15.0.11, 15.0, 15
```

Comment:
The `production` and `stable` tags are going away; was that on purpose?

Reply:
no, but it should be fixed now